client/src/components/PageTemplate/PageTemplate.tsx

```diff
@@ -1,21 +1,47 @@
 import React from 'react';
 import styled from 'styled-components';
+import { useLocation } from 'react-router-dom';
 import { Row, themeSpacing, themeColor } from '@amsterdam/asc-ui';
 import AppRoutes from '@app/AppRoutes';
 import * as AscLocal from '@components/LocalAsc/LocalAsc';
+import { Alert as AlertIcon } from '@amsterdam/asc-assets';
 import Footer from '@components/Footer/Footer';
+import content from '@services/content';
 
 interface IProps {
     className?: string;
 }
 
-const PageTemplate: React.FC<IProps> = ({ children, className }) => (
-    <StyledRow className={className}>
-        <AscLocal.Header fullWidth={false} tall homeLink={AppRoutes.HOMEPAGE.path} />
-        <StyledColumn span={12}>{children}</StyledColumn>
-        <Footer />
-    </StyledRow>
-);
+declare global {
+    interface Document {
+        documentMode?: any;
+    }
+}
+
+const isIE = /*@cc_on!@*/ false || !!document.documentMode;
+
+const PageTemplate: React.FC<IProps> = ({ children, className }) => {
+    const location = useLocation();
+    return (
+        <StyledRow className={className}>
+            <AscLocal.Header fullWidth={false} tall homeLink={AppRoutes.HOMEPAGE.path} />
+            <StyledColumn span={12}>
+                {isIE && location.pathname !== '/ie-support' && (
```

Comment:
In plaats van hardcoded /ie-support kun je hier ook de betreffende path uit AppRoutes gebruiken.